packages/block-library/src/file/index.js

```diff
@@ -77,20 +77,27 @@ export const settings = {
 		from: [
 			{
 				type: 'files',
-				isMatch: ( files ) => files.length === 1,
+				isMatch( files ) {
+					return files.length > 0;
```

Comment:
`isMatch` is required per https://github.com/WordPress/gutenberg/blob/fe51bd08684462690988de7ec61a3cdda083710a/packages/editor/src/components/block-drop-zone/index.js#L62 so this is just a simple sanity check.